src/XamlStyler.dotUltimate/src/rider/main/resources/META-INF/plugin.xml

```diff
@@ -0,0 +1,35 @@
+<idea-plugin>
+  <id>xamlstyler.rider</id>
+  <name>XAML Styler</name>
+  <version>_PLACEHOLDER_</version>
+  <vendor url="https://github.com/Xavalon/XamlStyler">Xavalon</vendor>
+  <idea-version since-build="_PLACEHOLDER_" until-build="_PLACEHOLDER_" />
+  <depends>com.intellij.modules.rider</depends>
+  <category>Formatting</category>
+
+  <description>
+<![CDATA[
+<p>XAML Styler is an extension that formats XAML source code based on a set of styling rules. This tool can help you/your team maintain a better XAML coding style as well as a much better XAML readability.</p>
+]]>
+  </description>
+
+  <extensions defaultExtensionNs="com.intellij">
+    <applicationConfigurable groupId="tools"
+                             instance="xavalon.plugins.xamlstyler.dotultimate.XamlStylerOptionsPage" />
+  </extensions>
+
+  <project-components>
+    <component>
+      <implementation-class>xavalon.plugins.xamlstyler.dotultimate.XamlStylerComponent</implementation-class>
+    </component>
+  </project-components>
+
+  <actions>
+    <action id="XamlStyler.Reformat"
```

Comment:
Do actions support custom icons?

Reply:
In the next commit, they do :-)

![image](https://user-images.githubusercontent.com/485230/81403931-bd641080-9134-11ea-88c1-d948d1df913d.png)

Reply:
(note the sub-level is not needed, invocation from the root level is possible as well)

Reply:
Nice! We _just_ added a [tiny icon style](https://github.com/Xavalon/XamlStyler/blob/master/src/XamlStyler.Extension.Windows/Resources/format.png) to help with contrast in context menus instead of the larger extension logo -- do you think this would work here as well? See #259.

Reply:
Updated. Picked a slightly different color style to match the XAML icon colors in Rider, makes it look nice 🤗

![image](https://user-images.githubusercontent.com/485230/82177915-17a26580-98db-11ea-83f2-e26b904cea81.png)